demos/smart_classroom_demo_gapi/include/stream_source.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <opencv2/videoio.hpp>
+#include <opencv2/gapi/garg.hpp>
+
+namespace cv {
+namespace gapi {
+namespace wip {
+class GCaptureSource : public IStreamSource
```

Comment:
We already have same class in out G-API framework, isn`t?

Reply:
Another heir with the same name (GCaptureSource)

Reply:
For TOP K rising hands case